src/main/java/teammates/ui/controller/InstructorFeedbackQuestionCopyAction.java

```diff
@@ -25,7 +25,7 @@ protected ActionResult execute() {
         try {
             int index = 0;
             String feedbackQuestionId = getRequestParamValue(Const.ParamsNames.FEEDBACK_QUESTION_ID + "-" + index);
-            statusToAdmin = "";
+            statusToAdmin.add("");
```

Comment:
This shouldn't be required - originally initialized to "" so that it can be appended to later.